src/test/regress/pg_regress_multi.pl

```diff
@@ -415,11 +415,15 @@ sub revert_replace_postgres
 # Avoid parallelism to stabilize explain plans
 push(@pgOptions, "max_parallel_workers_per_gather=0");
 
+# Help with debugging
+push(@pgOptions, "log_error_verbosity = 'verbose'");
```

Comment:
interesting that this didn't change any existing error verbosity, is this expected?

Reply:
This is for the logs, not the psql output.